web/src/components/base/Input.tsx

```diff
@@ -1,6 +1,13 @@
+import styled from '@emotion/styled'
 import TextField from '@mui/material/TextField'
 import React, { ChangeEvent, ReactElement } from 'react'
 
+const StyledTextField = styled(TextField)`
+  .MuiInputLabel-root {
+    z-index: 0;
+  }
```

Comment:
🙃 Could we change this to
```suggestion
  .${textFieldClasses.root} {
    z-index: 0;
  }
```

Reply:
My bad, sorry. Copy pasting is not always the best idea 🤦‍♂️